spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/utils/ConnectorUtil.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.spark.connector.utils;
+
+import static com.datastrato.gravitino.spark.connector.ConnectorConstants.COMMA;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+
+public class ConnectorUtil {
+
+  public static String removeDuplicates(String[] elements, String otherElements) {
```

Comment:
could you make the name of  `removeDuplicates` more meaningful like removeDuplicateSparkExtensions(String[] extensions, String[] addedExtensions)?

Reply:
> could you make the name of `removeDuplicates` more meaningful like removeDuplicateSparkExtensions(String[] extensions, String[] addedExtensions)?

ok

Reply:
updated.